README.md

```diff
@@ -10,6 +10,55 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs
 a single-node Kubernetes cluster inside a VM on your laptop for users looking
 to try out Kubernetes or develop with it day-to-day.
 
+## Requirements For Running Minikube
+* VirtualBox installation
+* VT-x/AMD-v virtualization must be enabled in BIOS
+
+## Usage
+
+Here's a brief overview of minikube usage:
+
+```shell
+$ minikube start
+Starting local Kubernetes cluster...
+Running pre-create checks...
+Creating machine...
+Starting local Kubernetes cluster...
+Kubernetes is available at https://192.168.99.100:443.
+
+$ $(minikube docker-env)
+$ docker build -t helloworld .
```

Comment:
As a user, I would wonder what this is... Is it a built-in thing or is it some required step or is it just a demo... and how may I try that demo out in that case...